web/src/message_view.ts

```diff
@@ -1145,6 +1145,7 @@ export function render_message_list_with_selected_message(opts: {
         // narrowing
         message_lists.current.view.set_message_offset(select_offset);
     }
+    message_lists.current.view.update_sticky_recipient_headers();
```

Comment:
Why is this the right place to add this?

Reply:
We want to call this after we have scrolled to the message we want to select.